modules/ssh/ssh.go

```diff
@@ -20,12 +20,18 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
-// Host is a remote host.
+// Host is a remote host, with some SSH options attached
 type Host struct {
-	Hostname    string // host name or ip address
+	*Options
+	Hostname string // host name or ip address
+}
+
+// SSH (connection) options, used for connecting to hosts
```

Comment:
NIT: This is confusing when there is also a `SshConnectionOptions` struct in the same module. Can we call this `AuthOptions` instead?

Reply:
Also, `CustomPort` probably belongs with `Host`

Reply:
@yorinasub17 valid feedback, and yea I was similarly torn re `SshConnectionOptions` and this struct. `AuthOptions` works4me, and I'll move `CustomPort` into `Host`. Give me a few mins